### PR TITLE
feat: ✨ Add api routes for suspending and unsuspending an User

### PR DESCRIPTION
**Pull request by LogischJo**
This Pull Request adresses the issue #359

### CODE REVIEW

File: app/Http/Controllers/Api/UserController.php [hunk 1]
```diff
@@ -181,6 +181,53 @@ public function decrement(Request $request, int $id)
...
+                    'error' => 'The user is already suspended',
+                ]);
+        }
+        $user->suspend();
+
+        return $user;
+    }
+
+    /**
+     * Suspends the user
```
Comment (BolverBlitz) [inline app/Http/Controllers/Api/UserController.php:208]: *Unsuspends

**Review by BolverBlitz — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Api/UserController.php (hunk 1 above)
Comment (LogischJo) [inline app/Http/Controllers/Api/UserController.php:208]: true lol

**Comment by 1day2die** [comment]
is this tested @BolverBlitz ?

**Comment by LogischJo** [comment]
It worked on my System using the at that time latest development version. But you should test it, because 2 tests are better than just one test.

**Comment by BolverBlitz** [comment]
Works on fresh dev branch pull + this pr.

**Pull request merged**